arkouda/client.py

```diff
@@ -959,10 +1003,6 @@ def get_mem_avail(unit: str = "b", as_percent: bool = False) -> int:
     """
     Compute the amount of memory available to be used.
 
-    Parameters
-     ----------
-     unit : str {'b', 'kb', 'mb', 'gb', 'tb', 'pb'}
-         unit of return ('b' by default)
```

Comment:
we prob wanna add this part back too

Reply:
@pierce314159 yeesh, that's no bueno, fixed!